src/features/view-nft/NFTDetailsCard/ui/CTAContainer/CTAContainer.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react';
+
+import { BuyNowButton } from '../../../../buy-now';
+
+import styles from './CTAContainer.module.scss';
+
+export interface CTAContainerProps {}
+
+export const CTAContainer: FC<CTAContainerProps> = ({}) => {
+	return (
+		<div className={styles.Container}>
+			<div className={styles.Content}>
+				<span className={styles.Label}>{'Buy Now (WILD)'}</span>
+				<div className={styles.Values}>
+					<span className={styles.TokenValue}>{'3,000,000.00'}</span>
```

Comment:
Hard coded values. The data will be added in a follow up PR. Aiming to separate image alignment task and updates to NFT CTA's